CHANGES.rst

```diff
@@ -422,6 +423,10 @@ astropy.tests
 
 - Updated top-level config file to properly ignore top-level directories.
   [#6449]
+- Fixed issue with running test suite directly from ``pytest``. [#6437]
+
+astropy.time
```

Comment:
I'll do a thorough review later. At a glance, the inclusion of extra ``astropy.time`` section seems unrelated.

Reply:
Oh yeah, that was definitely an oversight. It might have happened when I rebased the old PR on the latest `master` (using `git-rerere`!!!).

Reply:
Off topic: Someone needs to implement `git -argharghargh` to undo `rerere`.